tests/images/clusterloader2/Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM golang:1.16.4 AS builder
 WORKDIR /go/src/k8s.io
 RUN git clone https://github.com/kubernetes/perf-tests
 WORKDIR /go/src/k8s.io/perf-tests/clusterloader2
-RUN GOPROXY=direct go build -o ./clusterloader ./cmd
+RUN GOPROXY=direct GOOS=linux CGO_ENABLED=0  go build -o ./clusterloader ./cmd
 
-FROM amazon/aws-cli
+FROM alpine:3.15.4
```

Comment:
this change caused the clusterloader to fail as it requires awscli to run clusterloader2 tests for eks.